beamer/tests/contracts/test_request_manager_fees.py

```diff
@@ -200,19 +212,17 @@ def test_different_fees(request_manager, token, claim_period, claim_stake):
         request_id_1 = make_request(
             request_manager, token, requester, requester, amount, fee_data="standard"
         )
-        lp_fee_1 = request_manager.lpFee(token.address, amount)
+        lp_fee_1 = request_manager.lpFee(source_chain_id, target_chain_id, token.address, amount)
         protocol_fee_1 = request_manager.protocolFee(token.address, amount)
-        assert lp_fee_1 == 6e18
-        assert protocol_fee_1 == 63e15
+        assert lp_fee_1 > protocol_fee_1
 
     with temp_fee_data(request_manager, token, *fee_data_2):
         request_id_2 = make_request(
             request_manager, token, requester, requester, amount, fee_data="standard"
         )
-        lp_fee_2 = request_manager.lpFee(token.address, amount)
+        lp_fee_2 = request_manager.lpFee(source_chain_id, target_chain_id, token.address, amount)
         protocol_fee_2 = request_manager.protocolFee(token.address, amount)
-        assert lp_fee_2 == 36e15
-        assert protocol_fee_2 == 279e15
+        assert protocol_fee_2 > lp_fee_2
```

Comment:
Here, as above, we are checking for inequality, whereas before we checked for equality.
I am not sure whether we have another test that makes sure that the fees that are computed by the contract are exactly what they are supposed to be (i.e. the test that does the same calculation as the contract). But if we don't have such a test, we should have one, in order to catch changes to the fee model. I am not sure if this is such a test, but it feels weird that we are moving toward a more relaxed check here. Note: elsewhere it'd probably be fine because it's just a detail, but here we are in `test_request_manager_fees`, which is probably the only place that cares about fees.

Reply:
I think it wasn't me who changed this check. I have to look into this function.

Reply:
i changed it back to use exact values, i didnt want to calculate exact values before but it is ok now.